cmd/plan.go

```diff
@@ -140,6 +140,10 @@ func planHandler(cmd *cobra.Command, flags planFlags) {
 		return
 	}
 	logrus.Infof("Plan can be found at [%s].", planfile)
+	if len(p.Spec.Services) == 0 && len(p.Spec.InvokedByDefaultTransformers) == 0 {
+		logrus.Debugf("Plan : %+v", p)
+		logrus.Fatalf("Failed to find any services or default transformers. Aborting.")
```

Comment:
Do we need the Aborting word? Since it has actually finished the process.

Reply:
correct, as we are creating the plan file. Fixed it.